.github/workflows/terraform-plan-apply.yml

```diff
@@ -39,13 +41,23 @@ jobs:
               exit 1
             fi
           fi
-
           echo "Files changed since last commit..."
           echo "$FILES"
-
           # Escape newlines (replace \n with %0A)
           echo "::set-output name=diff::$( echo "$FILES" | sed ':a;N;$!ba;s/\n/%0A/g' )"
 
+      - name: Validate YML
+        id: validate_definiton_yml
+        env:
+          INPUT_SCHEMA: .ci/trial-json-schema.json
+        run: |
+          sudo npm install -g ajv-cli
+          export FILES=$(find . -name 'definition.yml')
+          for FILE in $(echo "$FILES" | sed "s/,/ /g")
+          do
+               ajv validate -s $INPUT_SCHEMA -d "$FILE"
+          done
```

Comment:
This can be written with the exec option of find:
https://stackoverflow.com/questions/9612090/how-to-loop-through-file-names-returned-by-find

Reply:
Fixed